### PR TITLE
Gravity manipulation field

### PR DESCRIPTION
**Pull request by Lucasje2**
https://zerofox.atlassian.net/browse/ROTP-107?atlOrigin=eyJpIjoiYTFkNDA4ZDY3MmJmNDIxMGE4MDE0MmJiMmZlOTJlMjciLCJwIjoiaiJ9

### CODE REVIEW

File: Assets/Scripts/GravityManipulationField/MatchParticleSystemSize.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            case GravityFieldController.GravityDirection.Down:
+                shapeModule.rotation = new Vector3(90, 0, 0);
+                break;
+            case GravityFieldController.GravityDirection.Left:
+                shapeModule.rotation = new Vector3(0, 0, -90);
+                break;
+            case GravityFieldController.GravityDirection.Right:
+                shapeModule.rotation = new Vector3(0, 0, 90);
+                break;
+        }
```
Comment (Extiriority) [inline Assets/Scripts/GravityManipulationField/MatchParticleSystemSize.cs:51]: Ditto

File: Assets/Scripts/GravityManipulationField/MatchParticleSystemSize.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            case GravityFieldController.GravityDirection.Down:
+                particleSystemRenderer.material = DownMaterial;
+                break;
+            case GravityFieldController.GravityDirection.Left:
+                particleSystemRenderer.material = LeftMaterial;
+                break;
+            case GravityFieldController.GravityDirection.Right:
+                particleSystemRenderer.material = RightMaterial;
+                break;
+        }
```
Comment (Extiriority) [inline Assets/Scripts/GravityManipulationField/MatchParticleSystemSize.cs:72]: Ditto

File: Assets/Scripts/GravityManipulationField/MatchParticleSystemSize.cs [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+                case GravityFieldController.GravityDirection.Down:
+                    shapeModule.scale = new Vector3(boxCollider.size.x, boxCollider.size.z, boxCollider.size.y);
+                    break;
+                case GravityFieldController.GravityDirection.Left:
+                    shapeModule.scale = new Vector3(boxCollider.size.z, boxCollider.size.x, boxCollider.size.z);
+                    break;
+                case GravityFieldController.GravityDirection.Right:
+                    shapeModule.scale = new Vector3(-boxCollider.size.z, boxCollider.size.x, -boxCollider.size.z);
+                    break;
+            }
```
Comment (Extiriority) [inline Assets/Scripts/GravityManipulationField/MatchParticleSystemSize.cs:95]: Statement has no default

**Review by Extiriority — CHANGES_REQUESTED** [review]
Try  to change the animal values on gravity force variable to about 3 so it floats more consistently
image

**Comment by Lucasje2** [comment]
This branch has been rejected because the malbers package has a gravity/wind field prefab that we can use

**Pull request closed without merging**